templates/ru/option-trailer.tmpl

```diff
@@ -1,5 +1,5 @@
 </TABLE>
 
-<P ALIGN="CENTER"><INPUT TYPE="SUBMIT" VALUE="Сохранить параметры"></P>
+<P ALIGN="CENTER"><INPUT TYPE="SUBMIT" VALUE="Установить параметры по умолчанию"></P>
 
-</DIV>
```

Comment:
What does this change mean? Looks unnecessary.

Reply:
Are you referring to the translation or to `</div>`?
The div is in the very bottom, and the newline has been removed from the file, that's why Github shows it that way (as is div has been removed and then added again).

This changes comes from @OlesyaGerasimenko commit, I did not change it.

Reply:
Fixed.

Reply:
I'm sorry for late response - there was a strange figure at the end of this part of code - like this https://tinyurl.com/mwhcskye , so it looked suspicious. Now I don't see it, so I'm happy, I guess :smile:

Reply:
All text-like files should end also the last line
with an explicit newline.
As far as I vaguely remember from ancient past
this is to ensure that traditional tools that process
text-like files line by line will also read the last line.
Tools may read always up to a newline character and when
there is no newline the last line could be missed.